include/triton/common/logging.h

```diff
@@ -73,9 +74,42 @@ class Logger {
   // Get the logging format.
   Format LogFormat() { return format_; }
 
+  // Get the logging format as a string.
+  std::string LogFormatString()
+  {
+    switch (format_) {
+      case Format::kISO8601:
+        return "ISO8601";
+      case Format::kDEFAULT:
+        return "default";
+      default:
+        return "";
```

Comment:
Do we want to return something when format is with unexpected value? If not, raise exception

Reply:
It is not possible to get this condition with the current code. The values are filtered prior to calling this function. It was only included because the compiler complained. Will an "Invalid format" just in case future changes impact this assumption.